exercises/acronym/cases_test.go

```diff
@@ -34,4 +34,8 @@ var stringTestCases = []acronymTest{
 		input:    "Complementary metal-oxide semiconductor",
 		expected: "CMOS",
 	},
+	{
+		input:    " Unusable   acronym over the years ",
+		expected: "UAOTY",
+	},
```

Comment:
This is a generated test. Its input is [here](https://github.com/exercism/problem-specifications/blob/master/exercises/acronym/canonical-data.json). 

So if it is a good idea to add this test case, it should happen on this parent repo.